asv_bench/benchmarks/benchmarks.py

```diff
@@ -11,159 +11,174 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+# define `MODIN_CPUS` env var to control the number of partitions
+# it should be defined before modin.pandas import (in case of using os.environ)
+
 import modin.pandas as pd
 import numpy as np
 
-from modin.config import CpuCount, TestDatasetSize
+from modin.config import TestDatasetSize
 from .utils import generate_dataframe, RAND_LOW, RAND_HIGH, random_string
 
-# define `MODIN_CPUS` env var to control the number of partitions
-# it should be defined before modin.pandas import
-pd.DEFAULT_NPARTITIONS = CpuCount.get()
+ASV_USE_IMPL = "modin"
 
 if TestDatasetSize.get() == "Big":
-    MERGE_DATA_SIZE = [
+    BINARY_OP_DATA_SIZE = [
         (5000, 5000, 5000, 5000),
-        (10, 1_000_000, 10, 1_000_000),
-        (1_000_000, 10, 1_000_000, 10),
+        # the case extremely inefficient
+        # (20, 500_000, 10, 1_000_000),
+        (500_000, 20, 1_000_000, 10),
     ]
-    GROUPBY_DATA_SIZE = [
+    UNARY_OP_DATA_SIZE = [
         (5000, 5000),
         (10, 1_000_000),
```

Comment:
`(10, 1_000_000)` case is also extremely inefficient, (10 rows, 1kk columns), groupby benchmarks with that size falls into a 60s timeout (with pandas and modin impl.)

Reply:
You are right.